resources/howtos/digitalocean_droplet.md

```diff
@@ -0,0 +1,109 @@
+# How to deploy a MeiliSearch instance on DigitalOcean
+
+## Create an out-of-the-box MeiliSearch
+
+### 1. Create a new droplet
+
+In any DigitalOcean page, when you are logged in, you will find a menu in the upper-right corner. Click on Create -> Droplets
```

Comment:
Added quotes because there are quotes for other buttons/tab in the tutorial. Doesn't have anything to do with the meaning.
```suggestion
In any DigitalOcean page, when you are logged in, you will find a menu in the upper-right corner. Click on "Create" -> "Droplets".
```